extensions/hibernate-reactive/deployment/src/main/java/io/quarkus/hibernate/reactive/deployment/HibernateReactiveProcessor.java

```diff
@@ -61,6 +97,194 @@ public void build(RecorderContext recorderContext,
         recorder.callHibernateReactiveFeatureInit(enableRx);
     }
 
+    @BuildStep
```

Comment:
This looks like largely copied from the blocking ORM processor. Could you add a comment to highlight what's similar and what's intentionally different?

Reply:
Sure thing, just pushed another commit with the comments